File: testprojects/scala_main/pipelines/LIVY_SCALA/code/src/main/scala/io/prophecy/pipelines/livy_scala/graph/Script_1.scala
package io.prophecy.pipelines.livy_scala.graph

import io.prophecy.libs._
import io.prophecy.pipelines.livy_scala.config.Context
import io.prophecy.pipelines.livy_scala.udfs.UDFs._
import io.prophecy.pipelines.livy_scala.udfs._
import io.prophecy.pipelines.livy_scala.udfs.PipelineInitCode._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Script_1 {
  def apply(context: Context, in0: DataFrame): DataFrame = {
    val spark = context.spark
    val Config = context.config
    var out0=in0
    out0
  }

}

File: testprojects/python_main/pipelines/PYTHON_GCP/code/pythondoanything/graph/Script_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from pythondoanything.config.ConfigStore import *
from pythondoanything.udfs.UDFs import *

def Script_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    concatstring = (
        str(Config.c_boolean)
        + str(Config.c_double)
        + str(Config.c_float)
        + str(Config.c_int)
        + str(Config.c_long)
        + str(Config.c_short)
        + str(Config.c_string1)
        + str(Config.c_repartition_expr)
        + str(Config.c_0)
    )
    print(f"Concatednated value is: {concatstring}")
    a = int(Config.c_1)
    b = int(Config.c_0)
    real_val = a * b
    assert (real_val == 0)
    out0 = in0

    return out0
